### ISSUE TITLE
Train with one file and query predictions from another file?

### ISSUE THREAD

**Issue by PabloMessina** [guoguibing/librec#54]
Hi!
I was wondering if it's possible to create a config file such that LibRec reads ratings from a file, say training.txt, and then I give it another file, say queries.txt, with only user_id's and item_id's, and then LibRec generates the rating predictions for those users over those items (i.e: answers those queries). I ask this question because as far as I understand LibRec reads a single file, and then it partitions the file into a training set and a test set, but in my case I would like to use the whole file for training and then make queries from another file. Note that I don't want to "test" with the other file, I just want to answer queries (in the queries.txt file I don't actually know the rating that user x would give to item y, so testing doesn't make sense in this context).

Looking forward to your response.

Thanks in advance.

**Comment by guoguibing** [comment]
Thanks for your suggestion. As a matter of fact, we are working on it for the next version of LibRec. We will add such a query to our LibRec other than evaluation.
